### PR TITLE
Big screens alignment

### PR DESCRIPTION
**Pull request by vauxhall**
Is not only for iPad, but also for Landscape on iPhone

### CODE REVIEW

File: Client/Frontend/Home/FirefoxHomeViewController.swift [hunk 1]
```diff
@@ -349,6 +349,16 @@ extension FirefoxHomeViewController {
                 let window = UIApplication.shared.keyWindow
                 let safeAreaInsets = window?.safeAreaInsets.left ?? 0
                 insets += FirefoxHomeUX.MinimumInsets + safeAreaInsets
+                
+                // Ecosia
+                if UIDevice.current.userInterfaceIdiom == .phone {
```
Comment (ecotopian) [inline Client/Frontend/Home/FirefoxHomeViewController.swift:354]: I think you can simplify this logic to:
`if UIDevice.current.userInterfaceIdiom == .pad || UIApplication.shared.statusBarOrientation.isLandscape {
  insets += frameWidth / 4
}`

So it does add insets when it's phone and portrait, right?

File: Client/Frontend/Home/FirefoxHomeViewController.swift [hunk 2]
```diff
@@ -974,6 +984,16 @@ class ASHeaderView: UICollectionReusableView {
 
     var titleInsets: CGFloat {
         get {
+            // Ecosia
+            
+            if UIDevice.current.userInterfaceIdiom == .phone {
```
Comment (ecotopian) [inline Client/Frontend/Home/FirefoxHomeViewController.swift:989]: same optimization for the boolean logic could apply here :)

**Review by ecotopian — APPROVED** [review]
Awesome! 🍀 just one suggestion.

File: Client/Frontend/Home/FirefoxHomeViewController.swift (hunk 1 above)
Comment (vauxhall) [reply Client/Frontend/Home/FirefoxHomeViewController.swift:354]: Good idea

**Review by ecotopian — APPROVED** [review]
💯

**Pull request merged**